doc/source/reference/routines.testing.rst

```diff
@@ -32,6 +29,13 @@ Asserts
    assert_warns
    assert_string_equal
 
+.. toctree::
+   :hidden:
+
+   assert_almost_equal
+   assert_approx_equal
+   assert_array_almost_equal
```

Comment:
If these are not recommended, can you add a `..note` in the docstring for each method that explains why, and what the alternative is?

Reply:
There is already a note in these functions pointing out to `assert_allclose` and alike. That's why I am listing these specifically.

Are you talking about something else?

Reply:
Also I am not sure how to hide these. Because if I am not mistaking they are only defined here so I would need to do the `autosummary`. But then I don't know how to hide.

Reply:
Ah, I was not aware of that.

Perhaps instead of hiding these you should just put them under a different heading, along with a remark saying that they are not recommended?

Reply:
OK sure. And also I am still bugged with how to do it otherwise 😅. So this? `Asserts (not recommanded)`